sdks/builds/ios.mk

```diff
@@ -297,17 +297,28 @@ $(eval $(call iOSSimulatorTemplate,sim64,x86_64))
 $(eval $(call iOSSimulatorTemplate,simtv,x86_64))
 $(eval $(call iOSSimulatorTemplate,simwatch,i386))
 
-LLVM_REV=3b82b3c9041eb997f627f881a67d20be37264e9c
+ifdef USE_PREBUILT_LLVM
```

Comment:
We have `IGNORE_PACKAGE_LLVM` for XA. It would be useful to use the same variable.